app-admin/dio/dio-1.5.2-r1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```

Comment:
why 6?

Reply:
Can I go to 7 directly? I read somewhere, that not all classes are ready for 7?

Reply:
Anyway, changed to `EAPI=7`, as it seems to work.